extensions/bitwarden/src/components/UnlockForm.tsx

```diff
@@ -94,7 +105,26 @@ const UnlockForm = ({ pendingAction = Promise.resolve() }: UnlockFormProps) => {
     >
       {shouldShowServer && <Form.Description title="Server URL" text={serverMessage} />}
       <Form.Description title="Vault Status" text={userMessage} />
-      <Form.PasswordField autoFocus id="password" title="Master Password" />
+      {showPassword ? (
+        <Form.TextField
+          autoFocus
+          id="plainPassword"
+          title="Master Password"
+          value={password}
+          onChange={setPassword}
+          ref={(field) => field?.focus()}
+        />
+      ) : (
+        <Form.PasswordField
+          autoFocus
```

Comment:
`autoFocus` can also be removed, it's redundant here.

```suggestion
          id="plainPassword"
          title="Master Password"
          value={password}
          onChange={setPassword}
          ref={(field) => field?.focus()}
        />
      ) : (
        <Form.PasswordField
```